extensions/amp-fit-text/1.0/example/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="/dist/bento.max.js"></script>
```

Comment:
i think we should use the full CDN sources for the bento.js scripts and the extension script/style

also i think we should use `bento.js` instead of `bento.max.js`